commit fc09363b2278893790d131c72a4ed96ec9837624
Author: Elly <ellyjones@chromium.org>
Date:   Thu Sep 21 22:09:59 2023 +0000

    a11y: don't redefine ATSPI version macros
    
    As of ATSPI 2.49.90, the library itself defines these, and if we
    redefine them on the compiler command line, we get a build warning.
    See: https://gitlab.gnome.org/GNOME/at-spi2-core/-/issues/129
    
    Bug: None
    Change-Id: I5cc655bd32c90d7af52c9dca60724f334c4a2a65
    Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/4883576
    Auto-Submit: Elly FJ <ellyjones@chromium.org>
    Reviewed-by: Thomas Anderson <thomasanderson@chromium.org>
    Commit-Queue: Thomas Anderson <thomasanderson@chromium.org>
    Cr-Commit-Position: refs/heads/main@{#1199870}

Index: chromium-119.0.6045.123/build/config/linux/atspi2/BUILD.gn
===================================================================
--- chromium-119.0.6045.123.orig/build/config/linux/atspi2/BUILD.gn
+++ chromium-119.0.6045.123/build/config/linux/atspi2/BUILD.gn
@@ -17,19 +17,13 @@ if (use_atk) {
                                       "--version-as-components",
                                     ],
                                 "value")
-    major = atspi_version[0]
-    minor = atspi_version[1]
-    micro = atspi_version[2]
-
-    # ATSPI 2.49.90 now defines these for us and it's an error for us to
-    # redefine them on the compiler command line.
-    # See ATSPI 927344a34cd5bf81fc64da4968241735ecb4f03b
-    if (minor < 49 || (minor == 49 && micro < 90)) {
-      defines = [
-        "ATSPI_MAJOR_VERSION=$major",
-        "ATSPI_MINOR_VERSION=$minor",
-        "ATSPI_MICRO_VERSION=$micro",
-      ]
-    }
+    atspi_major_version = atspi_version[0]
+    atspi_minor_version = atspi_version[1]
+    atspi_micro_version = atspi_version[2]
+    defines = [
+      "ATSPI_MAJOR_VERSION=$atspi_major_version",
+      "ATSPI_MINOR_VERSION=$atspi_minor_version",
+      "ATSPI_MICRO_VERSION=$atspi_micro_version",
+    ]
   }
 }
